catalog/ui/search-ui/search-endpoint/src/main/java/org/codice/ddf/ui/searchui/query/endpoint/CometdEndpoint.java

```diff
@@ -45,54 +43,39 @@
 import ddf.catalog.filter.FilterBuilder;
 
 /**
- * The CometdEndpoint binds the SearchService and the CometdServlet together. 
+ * The CometdEndpoint binds the SearchService and the CometdServlet together.
  * This is where the asynchronous endpoint is initially started.
  */
-public class CometdEndpoint {
+public class CometdEndpoint extends CometDServlet {
+    private static final long serialVersionUID = 1L;
 
     private static final Logger LOGGER = LoggerFactory.getLogger(CometdEndpoint.class);
 
-    private final CometDServlet cometdServlet;
+    private final transient FilterBuilder filterBuilder;
```

Comment:
Out of curiosity, why are any of these transient? I don't think the `CometdEndpoint` is serialized. Is intellij adding those?